src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -157,27 +187,11 @@ class FormPage extends React.Component {
           onSubmit={this.onSubmit}
         >
           {contentBeforeButtons}
-          <div className="row form-progress-buttons schemaform-buttons">
-            <div className="small-6 medium-5 columns">
-              {!isFirstRoutePage && (
-                <ProgressButton
-                  onButtonClick={this.goBack}
-                  buttonText="Back"
-                  buttonClass="usa-button-secondary"
-                  beforeText="«"
-                />
-              )}
-            </div>
-            <div className="small-6 medium-5 end columns">
-              <ProgressButton
-                submitButton
-                onButtonClick={callOnContinue}
-                buttonText="Continue"
-                buttonClass="usa-button-primary"
-                afterText="»"
-              />
-            </div>
-          </div>
+          <FormNavButtons
+            goBack={!isFirstRoutePage && this.goBack}
+            goForward={callOnContinue}
+            submitToContinue
```

Comment:
Is this prop really necessary? Looks like having `submitButton` hardcoded to `true` in `FormNavButtons` would eliminate the need for this.

Reply:
True, but that's not necessarily _how_ `CustomPage`s will use the navigation buttons. If, for example, the page doesn't have a `<form>` element, but just contains some text, having a `submit` button to continue to the next page won't work because the form submit action won't navigate. Instead, the `CustomPage` would have to render the continue button as a `<button type="button" onClick={goForward}>` (more or less) to navigate to the next page.